### DISCUSSION TITLE
Apparent Loss of Detail in Flux1 Dev from win_820 to win_0916

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [metercai/SimpleSDXL#76] [General]
I have been tracking the differences between the "win" and "win2" variants of SimpleSDXL 0820, in image sets with the same seeds, using Flux1 Dev. This started accidentally when I switched back to "win2" while troubleshooting a problem. I noticed that there was some lack of clarity in some win2 images, or a loss of detail. This only affected about 5% of images, the remaining 95% were identical.

With "win_0916" I have noticed quite significant differences in all the images I have tested so far. In most cases this is a loss of detail - the images are simpler. Please see the attachments.

Related to this, will there be a win2 version of "0916"?

![win2_0820 2024-09-11_19-38-39_6828](https://github.com/user-attachments/assets/78fef7c7-2280-415d-be04-cefcb81c577b)

![win_820 2024-09-13_10-51-55_1562](https://github.com/user-attachments/assets/dbd33180-b7e3-4306-a86c-0ea71808d79d)

![win_0916 2024-09-16_23-48-17_1914](https://github.com/user-attachments/assets/8bb5b17b-a1b7-450b-b501-10c2f98d4f34)

**Comment by metercai** [comment]
You need to track the cmd logs, as the model configuration parameters may change and their preset packages are not exactly the same. Win_820 is Fluxdev, while win_916 is Flux.

**Comment by DavidDragonsage** [comment]
The only differences between the Fluxdev and Flux presets are a modification to the "disinteractive": parameters and the removal of 	"task_method": "flux_base", from the 4th line. I tried Fluxdev with "win_0916" and the result is the same as with the Flux preset.

This is the log for "win_0820":
{
"prompt": "In Xanadu did Kubla Khan A stately pleasure-dome decree: Where Alph, the sacred river, ran Through caverns measureless to man Down to a sunless sea. So twice five miles of fertile ground With walls and towers were girdled round; And here were gardens bright with sinuous rills Where blossom'd many an incense-bearing tree; And here were forests ancient as the hills Enfolding sunny spots of greenery",
"negative_prompt": "",
"prompt_expansion": "",
"styles": "[]",
"performance": "Speed",
"steps": 20,
"resolution": "(1280, 960)",
"guidance_scale": 3.5,
"sharpness": 8,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "flux1-dev-bnb-nf4-v2.safetensors",
"refiner_model": "",
"refiner_switch": 1.0,
"sampler": "euler",
"scheduler": "simple",
"seed": "5848318787045970791",
"backend_engine": "Flux.1",
"metadata_scheme": "fooocus",
"version": "Fooocus v2.5.5 SimpleSDXL_v20240820.fe1ef95"
}

And this is the log for "win_0916":
{
"prompt": "In Xanadu did Kubla Khan A stately pleasure-dome decree: Where Alph, the sacred river, ran Through caverns measureless to man Down to a sunless sea. So twice five miles of fertile ground With walls and towers were girdled round; And here were gardens bright with sinuous rills Where blossom'd many an incense-bearing tree; And here were forests ancient as the hills Enfolding sunny spots of greenery",
"negative_prompt": "",
"prompt_expansion": "",
"styles": "[]",
"performance": "Speed",
"steps": 20,
"resolution": "(1280, 960)",
"guidance_scale": 3.5,
"sharpness": 8,
"adm_guidance": "(1.5, 0.8, 0.3)",
"base_model": "flux1-dev-bnb-nf4-v2.safetensors",
"refiner_model": "",
"refiner_switch": 1.0,
"sampler": "euler",
"scheduler": "simple",
"seed": "5848318787045970791",
"backend_engine": "Flux.1",
"metadata_scheme": "fooocus",
"version": "Fooocus v2.5.5 SimpleSDXL_v20240916.2952150"
}

Yes, that is the first thing I checked, but apart from the "version" listing the logs are identical. There is something else going on here.

I do not like the loss of detail, so I reinstalled "win_0820" for use with Flux1 Dev. Fortunately I installed a new NVMe SSD yesterday so I have plenty of space for both.

**Comment by DavidDragonsage** [comment]
This discussion item was transferred to the bug report: "Restore Detail to Flux1 Dev Image Generations #77", which is now closed.

The resolution was that "This is due to the difference caused by Comfyd upgrade, not a bug".

The higher detail in Flux1 Dev that is provided by win_820 is of course still available if the user chooses to use that  version.

@metercai  My remaining question is, is it practical to roll back the Comfy upgrade or are there other issues that make the Comfy upgrade necessary?
